package.json

```diff
@@ -23,6 +23,7 @@
     "@types/node": "^18.11.19",
     "@types/react": "18.0.27",
     "@types/react-dom": "18.0.10",
+    "clsx": "^2.0.0",
```

Comment:
a tiny package for cleaning up conditional class names; https://www.npmjs.com/package/clsx